resteasy-client-microprofile-base/src/main/java/org/jboss/resteasy/microprofile/client/RestClientBuilderImpl.java

```diff
@@ -20,6 +20,8 @@
 import org.jboss.resteasy.microprofile.client.header.ClientHeadersRequestFilter;
 import org.jboss.resteasy.microprofile.client.impl.MpClient;
 import org.jboss.resteasy.microprofile.client.impl.MpClientBuilderImpl;
+import org.jboss.resteasy.microprofile.client.publisher.MpPublisherMessageBodyReader;
+//import org.jboss.resteasy.microprofile.client.publisher.MpPublisherMessageBodyReader;
```

Comment:
Shouldn't we remove this commented line?

Reply:
Thanks. Fixed.